src/Microsoft.Experimental.Collections/Microsoft/Collections/Extensions/OrderedDictionary.KeyCollection.cs

```diff
@@ -11,14 +11,27 @@ namespace Microsoft.Collections.Extensions
 {
     public partial class OrderedDictionary<TKey, TValue>
     {
+        /// <summary>
+        /// Represents the collection of keys in a <see cref="OrderedDictionary{TKey, TValue}" />. This class cannot be inherited.
+        /// </summary>
         [DebuggerTypeProxy(typeof(DictionaryKeyCollectionDebugView<,>))]
         [DebuggerDisplay("Count = {Count}")]
         public sealed class KeyCollection : IList<TKey>, IReadOnlyList<TKey>
         {
             private readonly OrderedDictionary<TKey, TValue> _orderedDictionary;
 
+            /// <summary>
+            /// Gets the number of elements contained in the <see cref="OrderedDictionary{TKey, TValue}.KeyCollection" />.
+            /// </summary>
+            /// <returns>The number of elements contained in the <see cref="OrderedDictionary{TKey, TValue}.KeyCollection" />. Retrieving the value of this property is an O(1) operation.</returns>
```

Comment:
Should `Retrieving the value of this property is an O(1) operation.` be part of the summary tag?

Reply:
That's how it was specified in the Dictionary XML docs for its KeyCollection. I agree it would make more sense there.